subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -588,11 +588,18 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 		srv->delta_start = to_actual(status.current);
 	}
 
-	/* Clamp the value to the lightness range to prevent it be moved back to zero in binding
-	 * with Generic Level state.
-	 */
-	target_actual = CLAMP(srv->delta_start + delta_set->delta,
-			      srv->range.min, srv->range.max);
+	if (srv->delta_start == 0 && delta_set->delta < 0) {
```

Comment:
Shouldn't it be `<=`? If `delta_start` and `delta` are both zero, this will set lightness to range min.